compat/package.json

```
@@ -14,6 +14,7 @@
     "regex": "^_"
   },
   "peerDependencies": {
-    "preact": "^10.0.0"
+    "preact": "^10.0.0",
+    "preact-render-to-string": "^5.1.8"
```

Unsure if I should lower to 5.0.0
```suggestion
    "preact-render-to-string": "^5.0.0"
```

---

Maybe an optionalDependency is better? I don't know :)